package.json

```diff
@@ -7,21 +7,28 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@ariakit/react": "^0.2.3",
+    "@radix-ui/react-label": "^2.0.1",
```

Comment:
Are these packages in use?

Reply:
Not yet) As we still didn't define the library for components, I left them here to import and try the other elements from Ariakit and Radix. Anyway, I can delete them all.